tests/auth_tests/test_management.py

```diff
@@ -126,55 +126,47 @@ def tearDown(self):
 
     def test_that_changepassword_command_changes_joes_password(self):
         "Executing the changepassword management command should change joe's password"
-        self.assertTrue(self.user.check_password('qwerty'))
-        command = changepassword.Command()
-        command._get_pass = lambda *args: 'not qwerty'
+        with mock.patch.object(changepassword.Command, '_get_pass', return_value='not qwerty'):
```

Comment:
Looks like this patching could happen by decorating the method so we don't need indentation changes.